### DISCUSSION TITLE
What about Layr.JS?

### DISCUSSION THREAD

**Discussion by blarvin** [blitz-js/blitz#1812] [General]
@flybayer and everyone:

Burning question: How does BlitzJS compare to LayrJS (fka LiaisonJS)?

It looks like Blitz and Layr are very very similar. Is that true? How are they different? Which should I choose? Could these two projects learn from each other? (I intend to pester them with same question...)

(Context: I am a rather new dev who just wants to **build features**! I am being pulled in different directions by tools and technologies and methodologies! Also, from the very beginning of my journey, I have seen the need for exactly this: let me treat the back-end and front-end as part of a single whole. Good grief! So much glue and strings and popsicle sticks! ) 

Thank you for your work making life better! This is happening!

**Comment by Skn0tt** [comment]
Hi @blarvin! Haven't heard of Layr.js before. Had a quick look at the homepage, and these are (some) differences that sprung to my eye:

- Blitz.js makes full use of TypeScript (at least from the snippets on their webpage, it doesn't look Layr.js is easily typable)
- While Blitz.js allows you to use database library you want, Layr.js (seems to) lock you into their own `-store` libraries
- Layr.js has an unusual (but interesting) way of defining View Components

To me, one of the cool things about Blitz is that it builds on top of battle-proven stuff like Next.js and Prisma (at least by default, that is).
Layr.js seems like it's trying to do everything itself, which most definitely will slow down their development speed.

**Reply by blarvin** [reply]
Cool! 

I'll use your analysis to start putting together a comparison. Now there are 2!

Maybe there are more? 

You would agree these two projects have the same high-level goals, yes? 

For me it is boiling down to developer experience and ease of adoption. Of course. But then there will be a new system soon 🙃

**Reply by flybayer** [reply]
@blarvin take a look at http://redwoodjs.com for a true alternative to Blitz. Blitz and Redwood are two of kind right now. Nothing else really compares.
